streams/src/main/java/org/apache/kafka/streams/processor/internals/namedtopology/KafkaStreamsNamedTopologyWrapper.java

```diff
@@ -61,6 +74,8 @@ public KafkaStreamsNamedTopologyWrapper(final Properties props, final KafkaClien
 
     private KafkaStreamsNamedTopologyWrapper(final StreamsConfig config, final KafkaClientSupplier clientSupplier) {
         super(new TopologyMetadata(new ConcurrentSkipListMap<>(), config), config, clientSupplier);
+        final LogContext logContext = new LogContext();
```

Comment:
I think you can just instantiate the field like this instead of going through the `LogContext`, I checked that class and it seems to do the same thing as just
``` 
private final Logger log = LoggerFactory.getLogger(KafkaStreamsNamedTopologyWrapper.class);
```